icons/mailbox.svg

```diff
@@ -0,0 +1,16 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M11,19h9c1.1,0,2-0.9,2-2V9c0-2.2-1.8-4-4-4H6.5" />
+  <polyline points="15,9 18,9 18,11" />
+  <path d="M6.5,5L6.5,5C4,5,2,7,2,9.5V17c0,1.1,0.9,2,2,2h7V9.5C11,7,9,5,6.5,5z" />
+  <line x1="6" y1="10" x2="7" y2="10" />
```

Comment:
Nice one! A tiny thing though is that it'd be better to add the outline as one shape and the middle line as a separate path (should we ever implement support for filled/duotone icons).

```suggestion
  <path d="M22 17a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V9.5C2 7 4 5 6.5 5H18c2.2 0 4 1.8 4 4v8Z" />
  <polyline points="15,9 18,9 18,11" />
  <path d="M6.5 5C9 5 11 7 11 9.5V17a2 2 0 0 1-2 2v0" />
  <line x1="6" y1="10" x2="7" y2="10" />
```